### ISSUE TITLE
Regularization hyperparameters

### ISSUE THREAD

**Issue by wojtekkutak** [Kaszkietio/CINIC10_CNN_Classifier#7]
Tune hyperparameters related to regularization (must be 2), for example: batch normalization, dropout etc.

**Comment by wojtekkutak** [comment]
Done, results in dev branch.
